lerna.schema.json

```diff
@@ -0,0 +1,170 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
This is a generated schema, yes? Why does it need to be committed?

Reply:
Does it need to be published somewhere?

Reply:
It's been a little while so I don't remember, but I think I did this for code review purposes.